repo_policy_compliance/check.py

```diff
@@ -415,13 +415,13 @@ def execute_job(
             ),
         )
 
-    # Check that the commenter has maintain or above permissions
-    if not any(comment.user.login in maintain_logins for comment in authorization_comments):
+    # Check that the commenter has push or above permissions
```

Comment:
Can we keep the same vocabulary all along ? It should be either `push` permission or `write` permission

Reply:
So the thing is that GitHub uses the term interchangeably. On the API side they use `push`, in the UI they use `write` 
![Screenshot 2023-09-13 at 9 25 01 pm](https://github.com/canonical/repo-policy-compliance/assets/51036209/c197bca6-42b7-4a26-a3f1-457a7e2742a6)

I have adjusted it so that the code and tests use `push` and the user messages use `write` which I think makes the most sense for both contexts